hotel_roommatik/models/inherited_hotel_folio.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Jose Luis Algara (Alda hotels) <osotranquilo@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+import xml.etree.cElementTree as ET
+
+
+class Inherited_hotel_folio(models.Model):
```

Comment:
class HotelFolio(models.Model)
Class name without _ (UpperCamelCase) and without inherit